packages/ui5-test-writer/templates/v4/integration/opaTests.qunit.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
Tabs are used for indentation in this file. Can you please change them to spaces to be consistent with the other templates?

Reply:
done